tdrs-frontend/src/components/UploadReport/UploadReport.jsx

```diff
@@ -75,6 +78,7 @@ function UploadReport({ handleCancel, header, stt, submitEnabled }) {
         stt,
         uploadedFiles,
         user,
+        ssp: selectedFileType === 'ssp-moe',
```

Comment:
Minor nit, should we change this variable name to be consistent with `fileType` used elsewhere for this purpose? Would also need to be updated L176 in actions/reports.js I think

Reply:
not opposed

Reply:
Won't do, could be refactored later